install_config/web_console_customization.adoc

```diff
@@ -688,6 +688,51 @@ endif::[]
 ----
 
 endif::[]
+[[configuring-quota-notification-messages]]
+== Configuring Quota Notification Messages
+
+Whenever a user reaches or exceedes a quota a quota notification is put into the notificaiton drawer.
+A custom quota notification message, per
+xref:../dev_guide/compute_resources.adoc#dev-managed-by-quota[quota resource type], can be added to the notificaiton.  For example:
```

Comment:
s/notificaition/notification

Reply:
Hi @ahardin-rh, I'm not sure what you are asking to be addressed here?

Reply:
@dtaylor113 You have a typo in the spelling of "notification"